### PR TITLE
feat(CO-13) Frontend Hero Avatar

### PR DESCRIPTION
**Pull request by matyaspeterfi**

### CODE REVIEW

File: src/app/components/avatar/avatar.component.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="row">
+  <div class="card col-sm-2" (click)="navigateHero(hero.id, hero.name)">
```
Comment (juhaszPeterDev) [inline src/app/components/avatar/avatar.component.html]: dont define how the avatars should be presented on the frame
remove div with row and remove col-sm-2 leaving the rest of the div

File: src/app/components/avatar/avatar.component.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    <img src="{{hero.smallImage}}" class="card-img-top" *ngIf="hero">
+    <div class="card-footer">
+        <p class="card-title" *ngIf="hero">{{hero.name}} </p>
+    <p class="card-title finalWords" *ngIf="hero">"{{hero.finalWords}}"</p>
```
Comment (juhaszPeterDev) [inline src/app/components/avatar/avatar.component.html]: same ngIf. use it in the parent tag once

File: src/app/components/avatar/avatar.component.html (hunk 1 above)
Comment (matyaspeterfi) [reply src/app/components/avatar/avatar.component.html]: Fixed, cheers

File: src/app/components/avatar/avatar.component.html (hunk 2 above)
Comment (matyaspeterfi) [reply src/app/components/avatar/avatar.component.html]: fixed, cheers

**Pull request merged**